lib/solidus_dev_support.rb

```diff
@@ -10,5 +10,29 @@ def reset_spree_preferences_deprecated?
       first_version_without_reset = Gem::Requirement.new('>= 2.9')
       first_version_without_reset.satisfied_by?(Spree.solidus_gem_version)
     end
+
+    def cherry_picking_factories?
+      cherry_pick_factories_supported = Gem::Requirement.new('< 4.0.0.alpha').satisfied_by?(Spree.solidus_gem_version)
+      using_load_for = Pathname(Dir.pwd).join('spec/spec_helper.rb').read.include?('SolidusDevSupport::TestingSupport::Factories.load_for')
```

Comment:
This is brittle, as the code could be commented, but I can't see another option for that purpose...

Reply:
I've been experimenting with https://github.com/fxn/zeitwerk#the-on_load-callback, but I think it can't help here, as the required files won't define any constant.

Reply:
We could try with a regex but I guess it's super complex and has a lot of edge cases.

Reply:
Certainly, it's better to keep simple given there's no good solution.